packages/@okta/i18n/src/properties/login.properties

```
@@ -913,6 +913,13 @@ oie.custom_otp.description = Enter a temporary code generated from an authentica
 oie.custom_otp.authenticator.default.vendorName = Custom OTP Authenticator
 oie.custom_otp.verify.passcode.label = Enter code
 
+## Custom APP
+oie.verify.custom_app.title = Verify with {0}
+oie.custom_app.push.title = Get a push notification
+oie.custom_app.push.sent = Push notification sent
+oie.custom_app.push.resend = Resend push notification
+oie.custom_app.push.warning = Haven't received a push notification yet? Try opening the {0} App on your phone.
```

@akhtarshiekh-okta @panchamyadav-okta  The {0} here gets substituted with the custom authenticator name is that the expectation or we want to have a settings in authenticator for app name.

---

where is the custom authenticator name defined? how is it different from app name?

---

@magizh-okta - consider removing the `App` at the end of this sentence, as someone may have a custom authenticator named `Acme App`.

```suggestion
oie.custom_app.push.warning = Haven't received a push notification yet? Try opening {0} on your phone.
```

---

@panchamyadav-okta  the name of the authenticator is defined in the admin UI when creating the authenticator. 

The authenticator name can be either 'Wells fargo verify' or 'wells fargo verify app' depends on how admin names it. 
Depending on the name the text here might have 'app' repeated.

---

UX recommendation was 
'Haven't received a push notification yet? Try opening **Wells Fargo Authenticator** on your phone.'

---

Couldnt get the html embedded here and dint want to use  {{{ considering authenticator name is dynamic. UX was okay with not having bold.